embedder/embedder.go

```diff
@@ -184,6 +194,7 @@ func (flu *FlutterEngine) SendPointerEvent(event PointerEvent) Result {
 		y:              C.double(event.Y),
 		timestamp:      C.size_t(event.Timestamp),
 		signal_kind:    (C.FlutterPointerSignalKind)(event.SignalKind),
+		device_kind:    (C.FlutterPointerDeviceKind)(event.DeviceKind),
```

Comment:
how about setting `C.kFlutterPointerDeviceKindMouse`
Since we arn't currently supporting `C.kFlutterPointerDeviceKindTouch`